app/jobs/auto_enrol_job.rb

```diff
@@ -20,7 +20,7 @@ def perform(achievement:)
 
       Programmes::UserEnroller.new(
         programme_id: programme.id,
-        pathway_slug: pathway.slug,
+        pathway_slug: pathway&.slug,
```

Comment:
This change looks pragmatic and the user enroller is already set up to handle a nil pathway_slug 👍